src/Workspaces/CSharp/Portable/Formatting/Engine/Trivia/TriviaDataFactory.Analyzer.cs

```diff
@@ -59,7 +59,7 @@ public static AnalysisResult Between(SyntaxToken token1, SyntaxToken token2)
                     // newline.  If so, we keep track of that so we'll appropriately indent later
                     // on. 
 
-                    var previousNonMissingToken = token1.GetPreviousToken(includeZeroWidth: false, includeSkipped: true);
```

Comment:
Hmm - Are we sure we want to walk backwards past all skipped tokens trivia?  Seems like this TriviaData will not be what's intended anymore.  @heejaechang?